README.md

```diff
@@ -37,13 +37,15 @@ e.g Enable offline form submission with following settings for the forms availab
 
 ## Integration with izooto plugin.
 
+- The [iZooto](https://wordpress.org/plugins/izooto-web-push/) plugin must be installed and activated to use the izooto notification services.
 - Registers izooto service worker using `wp_front_service_worker` hook to main service worker file
 - Removes Service worker code of izooto from `?izooto=sw` to prevent conflicting it with main service worker.
```

Comment:
Is there any more descriptive steps needed to set up the iZooto?

Reply:
No, the user just needs to login with their izooto account with one click from izooto settings and details for which are already there in that setting page.